KsApi/models/CreateBackingEnvelope.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+public struct CreateBackingEnvelope: Decodable {
+  public var createBacking: CreateBacking
```

Comment:
Nevermind, looks like we need this since the structure of the response from graph is:

```
{
  "data": {
    "createBacking": {
      "clientMutationId": null,
      "checkout": {
        "state": "VERIFYING"
      }
    }
  }
}
```